Orange/evaluation/testing.py

```diff
@@ -366,21 +365,21 @@ def __init__(self, data, learners, store_data=False, store_models=False,
         self.call_callback(1)
 
 
-class Bootstrap(Results):
-    def __init__(self, data, learners, n_resamples=10, p=0.75, random_state=0,
-                 store_data=False, store_models=False, preprocessor=None,
-                 callback=None):
+class ShuffleSplit(Results):
+    def __init__(self, data, learners, n_resamples=10, p=0.75, test_size=None,
+                 random_state=0, store_data=False, store_models=False,
```

Comment:
I think we can leave the widget as it is.
I was referring to the class constructor.